build.gradle

```diff
@@ -37,10 +38,15 @@ repositories {
 
 ext {
     isSnapshot = "true" == System.getProperty("build.snapshot", "true")
+
```

Comment:
instead of hard coding buildVersionQualifier on top, it should be read from properties in here. if you add this change i can close #516 
```
buildVersionQualifier = System.getProperty("build.version_qualifier")
```